### PR TITLE
converted CI from travis to GHA

### PR DESCRIPTION
**Pull request by johnricords**

### CODE REVIEW

**Comment by johnricords** [comment]
the mockstack is failing; looks like a deprecation/versions issues with the version.tf experiments argument

**Comment by johnricords** [comment]
mockstacktest is now failing with an authentication error in the resource github_repository

**Comment by johnricords** [comment]
tested locally; successful
```
terraform-github-repo on  ghaConvert via 🐳 desktop-linux via 💠 default on ☁️  aws (us-east-1) 
✦ ❯ make terraform/pytest PYTEST_ARGS="-v --nomock"
[bootstrap]: Using external tardigrade-ci, skipping auto-init
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest] Starting Python tests found under the directory "/Volumes/Development/tardigrade-ci/tests/terraform_pytest"
pytest /Volumes/Development/tardigrade-ci/tests/terraform_pytest -v --nomock 
================================================ test session starts ================================================
platform darwin -- Python 3.10.9, pytest-7.2.1, pluggy-1.0.0 -- /Users/johnricords/.local/opt/python@3.10/bin/python3.10
cachedir: .pytest_cache
rootdir: /Volumes/Development
collected 1 item                                                                                                    

../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[create_repo] PASSED           [100%]

================================================ 1 passed in 20.25s =================================================
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest]: Tests executed!
```

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Note: this will only create an organizational repository
 At the moment, testing is manual:
 
 ```
-# Replace "xxx" with an actual AWS profile, then execute the integration tests.
+# Replace "xxx" with an your github access token, then execute the integration tests.
```
Comment (lorengordon) [inline README.md:12]: Good catch!

**Review by lorengordon — APPROVED** [review]
🚀

**Pull request merged**